### PR TITLE
Updating the Readme.MD

### PR DESCRIPTION
**Pull request by gcrev93**
No particular bug or issue created.

I am creating this pull request as a way to update the readme.

I updated many of the links, added a VS 2019 Installation section and removed some deprecated resources. 

Please note I want the users to come to me if they need installation for VS 2015 because this gives me a way to gain data on how many users are using older platforms with our tooling and if at any point we may need to update the extensions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,29 +7,34 @@ Node.js tools for Visual Studio 2017 is developed and managed here.
 
 
 Check out our documentation on [the NTVS wiki](https://github.com/Microsoft/nodejstools/wiki) 
-and our (old) [feature overview](https://channel9.msdn.com/events/Visual-Studio/Connect-event-2015/801) video on Channel 9.
+<!--and our (old) [feature overview](https://channel9.msdn.com/events/Visual-Studio/Connect-event-2015/801) video on Channel 9.
```
Comment (armanio123) [inline README.md]: I would remove any comments if there's no purpose for them.

File: README.md [hunk 2]
```diff
@@ -7,29 +7,34 @@ Node.js tools for Visual Studio 2017 is developed and managed here.
...
+-->
 
 Feel free to file issues or ask questions on our [issue tracker](http://github.com/Microsoft/nodejstools/issues), 
 and we welcome code contributions - see 
 [Contributing](https://github.com/Microsoft/nodejstools/blob/master/.github/CONTRIBUTING.md) for information. 
 
-## Visual Studio 2017 installation
+## Visual Studio 2019 installation
 
 The Node.js development workload is available as part of Visual Studio 2017 Community, Professional
```
Comment (armanio123) [inline README.md]: 2019?

File: README.md [hunk 3]
```diff
@@ -7,29 +7,34 @@ Node.js tools for Visual Studio 2017 is developed and managed here.
...
-## Visual Studio 2017 installation
+## Visual Studio 2019 installation
 
 The Node.js development workload is available as part of Visual Studio 2017 Community, Professional 
 and Enterprise. 
 To install, run the [normal VS installer](https://visualstudio.com/vs/downloads?wt.mc_id=github_microsoft_com) 
 and select the **Node.js development workload**.
 
-For the latest update to the Node.js tools install [Visual Studio Preview](https://www.visualstudio.com/vs/preview/).
+If you already have Visual Studio and only need to download the Node.js Tools, you can fine them [here](https://visualstudio.microsoft.com/downloads/?wt_mc_id=github_microsoft_com)
```
Comment (armanio123) [inline README.md]: Is this true? Node.js Tools should be installable as a workload on the Visual Studio Installer. I think we can point the user how to install the workload instead.

Comment (armanio123) [reply README.md]: Also, the link doesn't seem to point to anything related to Node.js

Comment (armanio123) [inline README.md]: Typo: `find` instead of `fine`.

File: README.md [hunk 4]
```diff
@@ -7,29 +7,34 @@ Node.js tools for Visual Studio 2017 is developed and managed here.
...
 
-## Visual Studio 2015 and earlier
 
-Prior to Visual Studio 2017, the Node.js tools were released as a standalone extension. We are no longer actively developing these versions, 
-but if you are unable to upgrade to Visual Studio 2017 yet, you can use these older installers to get the Node.js tools.
+## Visual Studio 2017 installation
 
-[NTVS 1.3.1](https://github.com/Microsoft/nodejstools/releases/tag/v1.3.1) is the latest available version for Visual Studio 2015.
+The Node.js development workload is available as part of Visual Studio 2017 Community, Professional
```
Comment (armanio123) [inline README.md]: nit: Seems like repetitive text. We might want to change the wording.

File: README.md (hunk 2 above)
Comment (gcrev93) [reply README.md]: yes..  the link for normal VS installer went out to VS 2019 download and when you download 2019 you can install the node tools. And I made a saw change as I saw 2017 was still in that description

File: README.md (hunk 3 above)
Comment (gcrev93) [reply README.md]: nope youre totally right!.. I read the description on them wrong

Comment (gcrev93) [reply README.md]: removing!

Comment (gcrev93) [reply README.md]: removing this line all together

File: README.md (hunk 1 above)
Comment (gcrev93) [reply README.md]: removed!

File: README.md (hunk 4 above)
Comment (gcrev93) [reply README.md]: any suggestions on wording? is it that we name all the versions that you dont like?

File: README.md (hunk 2 above)
Comment (armanio123) [reply README.md]: No, I mean the paragraph contains the text `2017`. I think we should change that to `2019` as well.

File: README.md (hunk 4 above)
Comment (armanio123) [reply README.md]: Maybe: "For Visual Studio 2017 Community, Professional and Enterprise, head to run the [2017 VS installer](https://visualstudio.microsoft.com/vs/older-downloads/) and choose the **Node.js development workload**. Please note you will need a Visual Studio Subscription to access this."

Does it looks better?

File: README.md (hunk 2 above)
Comment (gcrev93) [reply README.md]: Oh yes I fixed that in the new push!

File: README.md (hunk 4 above)
Comment (gcrev93) [reply README.md]: Maybe:

"For Visual Studio 2017 Community, Professional and Enterprise, run the 2017 VS installer and choose the Node.js development workload. Please note you will need a Visual Studio Subscription to access this"

File: README.md [hunk 5]
```diff
@@ -1,35 +1,35 @@
...
+and select the **Node.js development workload**.
 
-Prior to Visual Studio 2017, the Node.js tools were released as a standalone extension. We are no longer actively developing these versions, 
-but if you are unable to upgrade to Visual Studio 2017 yet, you can use these older installers to get the Node.js tools.
 
-[NTVS 1.3.1](https://github.com/Microsoft/nodejstools/releases/tag/v1.3.1) is the latest available version for Visual Studio 2015.
+## Visual Studio 2015 and earlier
 
-[NTVS 1.1](https://github.com/Microsoft/nodejstools/releases/tag/v1.1.1) is the last release for Visual Studio 2012, and 2013.
+For more information on how to install Node.js tools, please send an email to (Gabrielle Crevecoeur (Current Program Manager))[mailto:gacreve@microsoft.com?subject=Installing%20Node.js%20Tools%20For%20Early%20Versions%20%Of%20VS]
```
Comment (mrcrane) [inline README.md]: This mailto link is not rendering properly in the preview. I think you need `[]` around your name and `()` around the actual link.

Comment (gcrev93) [reply README.md]: Youre totally right. I mixed up my () and []

Comment (gcrev93) [reply README.md]: good catch !

File: README.md (hunk 4 above)
Comment (armanio123) [reply README.md]: Sounds good to me.

**Pull request merged**